validation/glean/baseline.2.core.pass.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "moz://mozilla.org/schemas/glean/ping/1",
```

Comment:
Quick flyby (the rest looks good to me): should this end with `/2`, given the version bump? Or it doesn't matter/is something else?

Reply:
Good catch.  You are right, this should be updated.